python/mxnet/gluon/contrib/estimator/event_handler.py

```diff
@@ -286,21 +293,31 @@ def train_end(self, estimator, *args, **kwargs):
         estimator.logger.info(msg.rstrip(', '))
 
     def batch_begin(self, estimator, *args, **kwargs):
-        if self.verbose == self.LOG_PER_BATCH:
+        if self.verbose >= self.LOG_PER_BATCH:
```

Comment:
this assumes the order of these constant variables. Better to use `if self.verbose in [var1, var2]` instead.

Reply:
I'll fix this.